src/main/java/games/strategy/engine/framework/lookandfeel/LookAndFeel.java

```diff
@@ -38,39 +87,4 @@ public static void setupLookAndFeel() {
     });
   }
 
-  private static String getDefaultLookAndFeel() {
```

Comment:
I removed this and now default all OS clients to ` UIManager.getSystemLookAndFeelClassName();` instead of ` SubstanceGraphiteLookAndFeel.class.getName();` for non-mac and  `UIManager.getSystemLookAndFeelClassName();`for mac.

I can restore that behavior if there is major objection, I suspect though we can just keep it simple. I can't think of a very strong reason for Mac OS to be special cased here.